code/game/turfs/open/floor/plating/asteroid.dm

```diff
@@ -122,6 +133,10 @@
 	planetary_atmos = TRUE
 	baseturfs = /turf/open/lava/smooth/cold
 
+/turf/open/floor/plating/asteroid/basalt/planetary
+	resistance_flags = INDESTRUCTIBLE
+	var/static/datum/gas_mixture/immutable/planetary
```

Comment:
Same here, I believe you need to set air equal to this value for this to do anything